install/linux/usr/share/odemis/sim/fastem-sim-asm.odm.yaml

```diff
@@ -319,19 +324,22 @@ FASTEM-sim: {
 "Diagnostic Camera": {
     class: simcam.Camera,
     role: diagnostic-ccd,
+    dependencies: {"focus": "Sample Stage in carrier coordinates"},
```

Comment:
I think it does not make a difference as z is not converted. Is there a preference?

Reply:
In Odemis, contrarily to X&Y, we don't have a strong convention about the Z direction. Normally, an increase in Z makes the lens and the sample closer. However, I wouldn't be surprised that in 50% of the systems that's wrong, and no one noticed (I'm only slightly exaggerating). Near the focus point, going closer or further away from the lens has almost the same effect, hence it's not easy to notice, and also it means the direction doesn't matter much.
So I'd say, you don't need to worry here. I think it makes a tiny bit more sense to use "Sample Stage in carrier coordinates" because *if* we cared, then that's the component where we could make sure the axis in the right direction.